src/cli.rs

```diff
@@ -45,11 +45,12 @@ pub(crate) fn parse_startup_args() -> Result<LdkUserInfo, ()> {
 		return Err(());
 	}
 	let bitcoind_rpc_info = env::args().skip(1).next().unwrap();
-	let bitcoind_rpc_info_parts: Vec<&str> = bitcoind_rpc_info.split("@").collect();
+	let mut bitcoind_rpc_info_parts: Vec<&str> = bitcoind_rpc_info.rsplitn(2,"@").collect();
 	if bitcoind_rpc_info_parts.len() != 2 {
 		println!("ERROR: bad bitcoind RPC URL provided");
 		return Err(());
 	}
+	bitcoind_rpc_info_parts.swap(0,1);
```

Comment:
nit: maybe more readable to just change the two lines that access `bitcoind_rpc_info_parts`?

Reply:
good suggestion, I tried this originally but wasn't sure if the reverse index access would be confusing, sounds like this is the better approach.